qemu/tests/uefishell.py

```diff
@@ -152,12 +175,13 @@ def run(test, params, env):
     """
 
     uefishell_test = UEFIShellTest(test, params, env)
+    time_interval = float(params.get("time_interval", 1))
```

Comment:
A global param, which always has a value '0.5', why not  params['time_interval']?

Reply:
agree with you. updated, thanks.